workitem/workitem_repository_blackbox_test.go

```diff
@@ -39,6 +39,14 @@ func (s *workItemRepoBlackBoxTest) SetupTest() {
 }
 
 func (s *workItemRepoBlackBoxTest) TestSave() {
+	s.T().Run("save work item without assignees & labels", func(t *testing.T) {
+		fxt := tf.NewTestFixture(t, s.DB, tf.WorkItems(1))
+		wi, err := s.repo.Save(s.Ctx, fxt.WorkItems[0].SpaceID, *fxt.WorkItems[0], fxt.Identities[0].ID)
```

Comment:
Don't just assume that the work item, created by your test fixture, has no labels or assignees. Please manually set assignees and labels to be empty.

Reply:
Fixed. 9392cc941712846c91f70d9a694da3a5c30acb4a